linters/elixir/.credo.exs

```diff
@@ -0,0 +1,21 @@
+%{
+  configs: [
+    %{
+      name: "default",
+      files: %{
+        included: ["lib/", "src/", "web/", "apps/"],
+        excluded: [~r"/_build/", ~r"/deps/"]
+      },
+      requires: [],
+      check_for_updates: true,
+      strict: true,
+      color: true,
+
+      checks: [
+        {Credo.Check.Design.TagTODO, exit_status: 0},
+        {Credo.Check.Readability.ModuleDoc, false},
```

Comment:
By default, `credo` requires a `@moduledoc` with a given model's description. This setting disables that check.